tutorials-v5/visualization/animation-demo.md

```diff
@@ -0,0 +1,146 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.13.8
+  kernelspec:
+    display_name: Python 3 (ipykernel)
+    language: python
+    name: python3
+---
+
+# Animation demos
+For more information about QuTiP see [http://qutip.org](http://qutip.org)
+
+
+## Overview
+QuTiP has animation functions to visualize the time evolution of quantum dynamics.
+
+
+```python
+from qutip import (ket, basis, sigmaz, tensor, qeye, mesolve,
+                   complex_array_to_rgb, about)
+import qutip
+import numpy as np
+import matplotlib.pyplot as plt
+```
+
+
+```python
+# a magic command enabling you to see animations in your jupyter notebook
+%matplotlib notebook
+```
+
+
+# Time evolution of a qubit
+Consider a system composed of two qubits. Its hamiltonian is $\sigma_z \otimes \mathbf{1}$ and the initial state is an entangled state ($\left|10\right>$+$\left|01\right>$)/$\sqrt2$.
+This operator acts on the first qubit and leaves the second qubit unaffected.
+
+
+```python
+# Hamiltonian
+H = tensor(sigmaz(), qeye(2))
+
+# initial state
+psi0 = (ket('10')+ket('01')).unit()
+
+# list of times for which the solver should store the state vector
+tlist = np.linspace(0, 3*np.pi, 100)
+
+results = mesolve(H, psi0, tlist, [], [])
+
+fig, ani = qutip.anim_schmidt(results)
+```
+
+
+The magic code may not work in your environments. This may happen if you run jupyter on Linux or use Google Colab. The code below will help you.
+
+
+```python
+# !pip install IPython
```

Comment:
If a user is running in a notebook, IPython should already be installed since it's a requirement to jupyter...

Reply:
Thanks for your info. I deleted it and actually use ```HTML(ani.to_jshtml())``` to make an animation.